apps/angular/src/core/dtos/anime.dto.ts

```diff
@@ -0,0 +1,54 @@
+// Because we get this type from backend
+/* eslint-disable @typescript-eslint/naming-convention */
+import { AnimeStatusDto } from './backend-enums/anime-status.dto';
+import { AnimeTypeDto } from './backend-enums/anime-type.dto';
+import { DateTimeRangeDto } from './date-time-range.dto';
+
+/** Represents anime in backend. */
+export type AnimeDto = {
+
+	/** Anime id. */
+	readonly id: number;
+
+	/**
+	 * Created.
+	 * @type {string} - String in ISO date format.
+	 */
+	readonly created: string;
```

Comment:
I think 'type' is redundant here since we have TypeScript in the project. Let's omit this:
```suggestion
	 * String in ISO date format.
	 */
	readonly created: string;
```

Reply:
Please, apply this change everywhere.